pkg/constants/constants.go

```diff
@@ -159,13 +159,22 @@ var (
 
 type AutoscalerClassType string
 type AutoscalerMetricsType string
+type AutoscalerKedaMetricsType string
 type AutoScalerKPAMetricsType string
 
 var (
 	AutoScalerKPAMetricsRPS         AutoScalerKPAMetricsType = "rps"
 	AutoScalerKPAMetricsConcurrency AutoScalerKPAMetricsType = "concurrency"
 )
 
+var (
+	AutoScalerMetricsAverageValue AutoscalerKedaMetricsType = "AverageValue"
+)
+
+var (
+	AutoScalerMetricsUtilization AutoscalerKedaMetricsType = "Utilization"
```

Comment:
You can put `AutoScalerMetricsUtilization AutoscalerKedaMetricsType = "Utilization"` together with `AutoScalerMetricsAverageValue` within one var block.
```
AutoScalerMetricsAverageValue AutoscalerKedaMetricsType = "AverageValue"
AutoScalerMetricsUtilization AutoscalerKedaMetricsType = "Utilization"
```